protocol/x/clob/keeper/orders.go

```diff
@@ -521,6 +522,12 @@ func (k Keeper) PlaceStatefulOrdersFromLastBlock(
 		}
 
 		order := orderPlacement.GetOrder()
+
+		// Prioritize placing post-only orders if the flag is set.
+		if onlyPlacePostOnly && !order.IsPostOnlyOrder() {
```

Comment:
We call this function twice, second time with `onlyPlacePostOnly` = false, which doesn't exclude the PO stateful orders. Does that mean PO orders are placed twice and is that an issue?